packages/react-components/react-drawer/src/components/DrawerInline/useDrawerInlineStyles.styles.ts

```diff
@@ -54,13 +53,8 @@ export const useDrawerInlineStyles_unstable = (state: DrawerInlineState): Drawer
   const baseClassNames = useDrawerBaseClassNames(state);
   const rootStyles = useDrawerRootStyles();
 
-  const separatorClass = React.useMemo(() => {
-    if (!state.separator) {
-      return undefined;
-    }
-
-    return state.position === 'start' ? rootStyles.separatorStart : rootStyles.separatorEnd;
-  }, [state.position, state.separator, rootStyles.separatorEnd, rootStyles.separatorStart]);
+  const separatorClass =
```

Comment:
**suggestion:** short circuiting is not the best pattern. if `separator` is falsy the value of separatorClass will be `false` instead of `string`